src/worker/keyman-api-service.ts

```diff
@@ -0,0 +1,36 @@
+import { keyboardDataObj, KeymanObj } from "./models";
+
+export class KeymanApi {
```

Comment:
Could this class be a single function? Classes with one method in addition to their constructor are candidates to be simplified to a simple function.

I can see the use of this class if it allowed for the dependency injection of `fetch()`, but the tests  overwrite `global.fetch` instead, so right now, it seems like a class is unnecessary.